### PR TITLE
Capitalize Haskell Stack in documentation

### PR DESCRIPTION
**Pull request by dschrempf**
See mailing list.

### CODE REVIEW

**Comment by tomjaguarpaw** [comment]
> See mailing list.

Specifically <https://mail.haskell.org/pipermail/haskell-cafe/2022-April/135214.html>

**Review by tomjaguarpaw — APPROVED** [review]

**Comment by tomjaguarpaw** [comment]
Thanks! Our policy requires two committee approvals before merging so we'll have to wait for another review.

**Review by TikhonJelvis — APPROVED** [review]
Looks great. Thanks for the fix!

**Pull request merged**
